nilearn/experimental/surface/_surface_image.py

```diff
@@ -4,10 +4,15 @@
 
 import abc
 import pathlib
+import sys
+from pathlib import Path
 
+import nibabel as nb
```

Comment:
```suggestion
import nibabel as nib
```

Reply:
actually we have a bit of mix of both in the code base: will do a quick clean up